python/sdk/prelude_sdk/models/codes.py

```diff
@@ -260,6 +261,7 @@ class NotificationTypes(Enum, metaclass=MissingItem):
     NEW_MISSING_EDR_POLICY_ENDPOINTS = 3
     NEW_MISSING_AV_POLICY_ENDPOINTS = 4
     NEW_MISSING_MFA_USERS = 5
+    NEW_MISSING_ASSET_MANAGER = 6
```

Comment:
To be synonymous with `NEW_NO_EDR_ENDPOINTS`, should we name this `NEW_NO_ASSET_MANAGER_ENDPOINTS`?  i.e.: change "missing" to "no" & add "endpoints" as a suffix?

Reply:
good idea thanks!